File: Text/check_palindrome/check_palindrome.cpp
#include <iostream>
int main() {
    std::string str;
    std::cout << "Word to check for palindrome: ";
    std::cin >> str;
    for (int i = 0, len = str.length(), j = len - 1; i < j; i++, j--) {
        if (str[i] != str[j]) {
            std::cout << "Is not a palindrome";
            return 0;
        }
    }
    std::cout << "Is a palindrome";
}

File: Text/check_palindrome/check_palindrome.js
// check if the word is a palindrome 

function Palindrome(word){ 
    let reverse = word.split('').reverse().join('');
    if (word == reverse){
        console.log(word + " is a palindrome");
    }
    else{
        console.log(word + " is not a palindrome");
    }
}

// is a palindrome
Palindrome("mom");

// is not a palindrome
Palindrome("hello");


